containers/ecr-viewer/src/app/utils/evaluate/fhir-paths.ts

```diff
@@ -350,6 +353,21 @@ const _fhirPathMappings: { [K in FhirPathKeys]: Omit<FhirPath<K>, "name"> } = {
     type: "Identifier",
     path: "Bundle.entry.resource.where(resourceType = 'Encounter')[0].identifier",
   },
+
+  hospitalEncounterDiagnosisRef: {
+    type: "Reference",
+    path: "Bundle.entry.resource.where(resourceType = 'Composition').section.where(code.coding.where(code = %code).exists()).entry",
```

Comment:
```suggestion
    path: "Bundle.entry.resource.where(resourceType = 'Composition').section.where(code.coding.code = %code).entry",
```

is this equivalent? the exists threw me a bit on readability

Reply:
It is equivalent, swapped to that. Thanks!